pages/manage_and_operate/iam/identities-management/guide.en-asia.md

```diff
@@ -22,7 +22,7 @@ There are several types of identities that can interact with OVHcloud products:
 
 ### OVHcloud account
 
-This is the primary identity used to log in to the OVHcloud Control Panel. You use the OVHcloud account when you log in to the OVHcloud Control Panel with your email address.
+This is the primary identity used to log in to the OVHcloud Control Panel. You use the OVHcloud account when you log in to the OVHcloud Control Panel with your email address or NIC handle (e.g. xx1111-ovh).
```

Comment:
@gbarideau AFAIK the concept of NIC handle is not used in CA/APAC. Customers use their main email address to log in to their account, no NIC is displayed in their account either.